remote-workstations/new-agent-vm/Install-PCoIPAgent.sh

```diff
@@ -102,7 +104,19 @@ sudo yum -y install sssd realmd oddjob oddjob-mkhomedir adcli samba-common samba
 sudo systemctl enable sssd
 
 echo "-->Joining the domain"
-echo "$PASSWORD" | sudo realm join --user="$USERNAME" "$DOMAIN_NAME"
+if [ ${OU} ]
```

Comment:
Curiosity - so when should we use if [ ${OU} ] versus if [ -n ${OU} ] or should it be if [ -n "${OU}" ] ? (I have a leaning for the latter based on some examples I saw but I don't really know why.)

Reply:
yeah, I think the latter is safer. mine might break if it's an int or something unexpected like that